simple_analog_clock.py

```diff
@@ -26,15 +26,15 @@ def draw_clock(h, m, s, pen): # Defining function, draw a clock, using pen turtl
     pen.up() # Dont want to draw a line
     pen.goto(0, 210) # Placement of pen tool
     pen.setheading(180) # turtle facing to the eft
-    pen.color("green") # Color
```

Comment:
@LukeLorenger is this where I should change the color?

Reply:
Yes, that is where you would change the color. Nice job.